src/public/Interfaces.ts

```diff
@@ -1,54 +1,157 @@
+/**
+ * Interface for your Azure Function code. This function must be exported (via module.exports or exports) 
+ * and will execute when triggered. It is recommended that you declare this function as async, which will 
+ * implicitly returns a Promise.
+ * @param context IContext object passed to your function from the Azure Functions runtime.
+ * @param {any[]} args Optional array of input and trigger binding data. These binding data are passed to the 
+ * function in the same order that they are defined in function.json.
+ * @returns Output bindings (optional).
+ */
+export interface IFunction {
+    (context: IContext, ...args: any[]): Promise<any> | void;
+}
+
+/**
+ * The context object can be used for writing logs, reading data from bindings, setting outputs and using 
+ * the context.done callback when your exported function is declared synchronous. A context object is passed 
```

Comment:
Grammatically, I feel like this should be either "function is declared synchronously." or "function is declared as synchronous." Just a nit and I could be wrong

Reply:
Yeah, agreed :) Thoughts on "your exported function is synchronous"?